src/lib/Menu/Admin/Role/RoleCreateRightSidebarBuilder.php

```diff
@@ -52,16 +28,13 @@ protected function getConfigureEventName(): string
     /**
      * @param array $options
      *
-     * @return \Knp\Menu\ItemInterface
-     *
      * @throws \InvalidArgumentException
      * @throws ApiExceptions\BadStateException
      * @throws \InvalidArgumentException
      */
     public function createStructure(array $options): ItemInterface
     {
-        $saveId = $options['save_id'];
-        $saveAncCloseId = $options['save_and_close_id'];
+        $saveAndCloseId = $options['save_and_close_id'];
```

Comment:
Should it be `save_and_close_id` as the button is about creating?

Reply:
they are all named like this, for update as well, sometimes they are named `save_and_edit`, I really do not want to change all of them now. Even more that technicaly is BC break.